Sophos/sophos firewall/_meta/smart-descriptions.json

```
@@ -0,0 +1,135 @@
+[
+  {
+    "value": "{user.name} connected to {destination.registered_domain} over {network.transport}",
+    "conditions": [
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "destination.registered_domain"
+      },
+      {
+        "field": "source.port"
+      },
+      {
+        "field": "network.transport"
+      }   
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "destination.registered_domain",
+        "type": "connected to"
+      }
+    ]
+  },
+  {
+    "value": "{destination.ip} {sophos.action} {sophos.sub} from {source.ip}",
```

```suggestion
    "value": "Action type {sophos.action} identified by component {sophos.sub} between {source.ip} and {destination.ip}:{destination.port}",
```

Since it is the same as the one below I suggest using the same sentence :point_down: